charts/admission/charts/application/templates/rbac.yaml

```diff
@@ -5,7 +5,15 @@ metadata:
   name: {{ include "name" . }}
   labels:
 {{ include "labels" . | indent 4 }}
-rules: []
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Why you need `list` and `watch` permissions when you only have apiReader that does get requests for secrets?